src/functionalTest/resources/features/F-003/S-022.td.json

```diff
@@ -39,6 +39,25 @@
             "workTypes" : "[[ANYTHING_PRESENT]]"
           },
           "authorisations" : "[[ANYTHING_PRESENT]]"
+        },
```

Comment:
can we add sort criteria to the start of the array: 
```json 
{
        "__elementId__": "roleName"
},
```

Reply:
addressed in c94c50b